lib/msal-common/docs/Accounts.md

```diff
@@ -8,55 +8,20 @@ export type AccountInfo = {
     environment: string;
     tenantId: string;
     username: string;
+    localAccountId: string;
+    name?: string;
 };
 ```
 
-## Usage
+## Account retrieval APIs
```

Comment:
Can we add description for what is `homeAccountId` and `localAccountId`?

Reply:
I added a section for each, please let me know if everything looks right or if corrections/additions are necessary!